enclave/src/lib.rs

```diff
@@ -1031,12 +1041,7 @@ fn handle_ocex_withdraw(
     }
 }
 
-fn execute_ocex_release_extrinsic(
-    acc: AccountId,
-    token: AssetId,
-    amount: u128,
-    mut nonce: u32,
-) -> SgxResult<()> {
+fn execute_ocex_release_extrinsic(acc: AccountId, token: AssetId, amount: u128) -> SgxResult<()> {
     // TODO: compose an ocex::release extrinsic, sign with enclave signing key and send it through ocall
     let mut validator = match io::light_validation::unseal() {
         Ok(v) => v,
```

Comment:
Do we need to unseal validator and do this error-handling stuff and unwrap() every time because genesis hash is a constant value, and we can load it once to memory to an atomic pointer and read from it? We don't even need mutex since it's read-only.

Reply:
This can be done later